### PR TITLE
docs(sample): get sink metadata

### PR DESCRIPTION
**Pull request by athakor**

### CODE REVIEW

File: samples/snippets/src/test/java/com/example/logging/SinkIT.java [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+  public void tearDown() {
+    // Delete sink
+    logging.deleteSink(SINK_NAME);
+  }
+
+  @Test
+  public void testGetSinkMetadata() throws Exception {
+    Sink sink = logging.getSink(SINK_NAME);
+    PrintStream standardOut = System.out;
+    final ByteArrayOutputStream snippetOutputCapture = new ByteArrayOutputStream();
```
Comment (suraj-qlogic) [inline samples/snippets/src/test/java/com/example/logging/SinkIT.java]: final modifier is mandatory here?

Comment (athakor) [reply samples/snippets/src/test/java/com/example/logging/SinkIT.java]: no, not mandatory. done

**Comment by athakor** [comment]
build failure unrelated to this changes

File: samples/snippets/src/test/java/com/example/logging/SinkIT.java [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.junit.runners.JUnit4;
+
+/** Tests for sink sample. */
+@RunWith(JUnit4.class)
+public class SinkIT {
+
+  private Logging logging = LoggingOptions.getDefaultInstance().getService();
```
Comment (suraj-qlogic) [inline samples/snippets/src/test/java/com/example/logging/SinkIT.java:39]: cleanup resources of logging in tear down

Comment (athakor) [reply samples/snippets/src/test/java/com/example/logging/SinkIT.java:39]: done

**Review by suraj-qlogic — APPROVED** [review]

**Pull request closed without merging**